strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/proxied/LocalStorageProxyRepositoryExpiredArtifactsCleaner.java

```diff
@@ -41,74 +42,70 @@
     private RepositoryPathResolver repositoryPathResolver;
 
     @Inject
-    private ArtifactEntryService artifactEntryService;
+    private ArtifactRepository artifactEntityRepository;
 
     @Inject
     private RemoteRepositoryAlivenessService remoteRepositoryAlivenessCacheManager;
 
     @Inject
     private ArtifactManagementService artifactManagementService;
 
-    @Transactional(rollbackFor = Exception.class)
+    @Transactional
     public void cleanup(final Integer lastAccessedTimeInDays,
                         final Long minSizeInBytes)
             throws IOException
     {
-        final ArtifactEntrySearchCriteria searchCriteria = anArtifactEntrySearchCriteria()
-                                                                   .withLastAccessedTimeInDays(lastAccessedTimeInDays)
-                                                                   .withMinSizeInBytes(minSizeInBytes)
-                                                                   .build();
-
-        final List<ArtifactEntry> artifactEntries = artifactEntryService.findMatching(searchCriteria,
-                                                                                      PagingCriteria.ALL);
-        filterAccessibleProxiedArtifacts(artifactEntries);
-
-        if (CollectionUtils.isEmpty(artifactEntries))
+        final Page<Artifact> artifactEntries = artifactEntityRepository.findMatching(lastAccessedTimeInDays, minSizeInBytes,
+                                                                                     PageRequest.of(0, Integer.MAX_VALUE));
```

Comment:
Cosmetic: One parameter per line when multi-line.
```suggestion
        final Page<Artifact> artifactEntries = artifactEntityRepository.findMatching(lastAccessedTimeInDays,
                                                                                     minSizeInBytes,
                                                                                     PageRequest.of(0, Integer.MAX_VALUE));
```